packages/login/src/components/LoginForm/LoginForm.js

```diff
@@ -61,6 +63,7 @@ export class LoginForm extends Component {
                 onChange={this.handlePasswordChange.bind(this)}
                 placeholder={this.msg('client.login.form.passwordPlaceholder')}
                 required
+                autoFocus={passwordFocus}
```

Comment:
`autoFocus` is still set on username field. Maybe it would be better to set it explicitly only if the password field shouldn't be focused.